protostar/commands/init/init_command.py

```diff
@@ -17,11 +18,13 @@ def __init__(
         requester: InputRequester,
         new_project_creator: NewProjectCreator,
         adapted_project_creator: AdaptedProjectCreator,
+        cwd: Path,
     ) -> None:
         super().__init__()
         self._adapted_project_creator = adapted_project_creator
         self._new_project_creator = new_project_creator
         self._requester = requester
+        self._cwd = cwd
```

Comment:
I don't understand why this code is passing `cwd` parameter everywhere. Wouldn't setting global cwd upon Protostar starting be more fault proof? As Protostar developer, I would expect cwd to be set to project root entire Protostar execution time.

Reply:
Resolving, removed cwd. Modified `ProtostarFixture::init` instead.